src/components/Tooltip/Tooltip.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'react-emotion';
+
+import { textKilo } from '../../styles/style-helpers';
+
+const baseStyles = css`
+  label: tooltip;
+  position: relative;
+  display: inline-block;
+
+  &:hover {
+    > span {
+      visibility: visible;
+      opacity: 1;
+    }
+  }
+`;
+
+const arrowAligns = theme => ({
+  center: `left: 50%`,
+  right: `left: ${theme.spacings.mega}`,
+  left: `right: ${theme.spacings.mega}`
+});
+
+const tooltipAlign = {
+  center: `
+    left: 0;
+    right: 0;
+    margin-left: auto;
+    margin-right: auto;
+  `,
+  right: 'left: 50%;',
+  left: 'right: 50%;'
+};
+
+const tooltipTextStyles = ({ align, theme }) => css`
+  label: tooltip_text;
```

Comment:
Don't you need to have something like `label: tooltip__text` and then for the alignment, `label: tooltip__text--${align}`?

Reply:
Yeah. Connor told me about this pattern on the last PR :) 
Let me change